ocaml/src/asd_server.ml

```diff
@@ -581,6 +581,71 @@ let execute_query : type req res.
                         return
                           ~cost:(200 * List.length keys)
                           res)
+    | PartialGet ->
+       fun (key, slices, prio) ->
+       begin
+         match get_value_option kv key with
+         | None -> return' false
+         | Some (_cs, blob) ->
+            let cost =
+              List.fold_left
+                (fun acc (_, length) ->
+                 acc + 200 + length)
+                0
+                slices
+            in
+            perform_read
+              io_sched
+              prio
+              (fun () ->
+               Lwt_log.ign_debug_f "PartialGet for %s" (Slice.show key);
```

Comment:
I copy pasted that from the MultiGet(2) implementation, but it is indeed not needed, so removed the ign_.